mantidproject/mantidimaging
# Handles missing projection or projection angle

Verifies the number of images versus the number of proj angles in log 

- If they mismatch it raises a runtime error
- It tries to find the missing projection / image file

It shows an error message and should not select the log. The error message ought to prompt the user to inspect their data and/or logs and fix the change (we can't really do anything!)

### To test
- The flower dataset should have logs in it
- Copy your image files and the sample log, as you'll be editing those and don't want to lose the main copy
- Delete any image file 
- Try loading the data - it should tell you there's a missing image file (projection), and do a best guess as to which one (number might be off by 1 for the image file)
  - Edge cases are: missing first or last image file, try those too

---------------

- Restore the image file and delete a line from the log
- Try loading the data again - this time it should tell you there's a missing angle
  - Same edge case as above

---------------

- Keep the deleted line in the log and delete A DIFFERENT image file 
- Try loading the data - it should fail again

---------------

- Restore the previous projection, keep the deleted line in the log and delete the SAME image file as the line in the log
- This should now load as there is no mismatching information
